client/src/js/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
+import AppStore from '../../stores/AppStore';
+import NavDash from './NavDash';
+import LHS from './Lhs';
+import MessageBoard from './MessageBoard'
+import DisplayMessage from './DisplayMessage'
+
+
+import {Grid, Row, Col, Clearfix} from 'react-bootstrap';
+
+export default class DashBoard extends Component {
+      constructor(props){
+        super(props);
+        this.state ={
+           authed: false,
+            user : AppStore.getUser(),      
+            contacts: AppStore.getGroupUsers(),
+            emails: AppStore.getGroupEmails(),
+            numbers:AppStore.getGroupNumbers(),
+            groups: AppStore.getGroups(),
+            currentGroup: AppStore.getCurrentGroup(),
+            databaseUsers: AppStore.getdatabaseUsers(),
+            notification: AppStore.getNotification(),
+            displayArchives: AppStore.getOpenArchive()  
+        };
+         this._onChange= this._onChange.bind(this)
+    }
+
+   componentWillMount ()  {
+     AppStore.addChangeListener(this._onChange);
+  }
+
+  componentWillUnmount () {
+    AppStore.removeChangeListener(this._onChange);
+  } 
+
+  render() { 
+    return (
+      <div id='dash'>
+        <NavDash contact={this.state.contacts} group={this.state.groups} user={this.state.user} databaseUsers={this.state.databaseUsers} notification={this.state.notification}/>  
```

Comment:
Use multiple lines:

> <NavDash
  contact={this.state.contacts}
  group={this.state.groups}
  user={this.state.user}
  databaseUsers={this.state.databaseUsers}
  notification={this.state.notification}
/>